website/src/redux/reducers/ReducerConstants.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file Constants needed for the Redux reducers.
```

Comment:
for this file I would look at the firebase schema and try and match the item names so that it is more clear what item in redux corresponds to what item in firebase. Ex: the list of users on a canvas in firebase is called users, but here it is called collaborators. Could be confusing for someone who is trying to debug